doc/source/whatsnew/v0.19.0.rst

```diff
@@ -164,13 +158,13 @@ Using a non-regular, but still monotonic index, rolling with an integer window d
 .. ipython:: python
 
 
-   dft = DataFrame({'B': [0, 1, 2, np.nan, 4]},
-                   index = pd.Index([pd.Timestamp('20130101 09:00:00'),
-                                     pd.Timestamp('20130101 09:00:02'),
-                                     pd.Timestamp('20130101 09:00:03'),
-                                     pd.Timestamp('20130101 09:00:05'),
-                                     pd.Timestamp('20130101 09:00:06')],
-                                    name='foo'))
+   dft = pd.DataFrame({'B': [0, 1, 2, np.nan, 4]},
```

Comment:
here too

Reply:
I think I realigned all to overlap with the first letter "i" or "c". Let me know if I miss any block.